dev/reference/tools/data-migration/manage-tasks.md

```diff
@@ -16,14 +16,17 @@ category: reference
 ```bash
 $ ./dmctl --help
 Usage of dmctl:
- # 打印版本信息
  -V prints version and exit
  # 按照 DM 提供的加密方法加密数据库密码，用于 DM 的配置文件
```

Comment:
This line should be before L23

Reply:
fixed in https://github.com/pingcap/docs-cn/pull/1878/commits/e234e6af6dd8a31ac85a4b1578b67292e9b97ca3.